csharp/src/Ice/MetricsObserver.cs

```diff
@@ -86,7 +86,7 @@ internal void ForEach(MetricsUpdate u)
             }
         }
 
-        internal MetricsMap<T>.Entry? GetEntry(MetricsMap<T> map) => _entries.FirstOrDefault(entry => entry.Map == map);
+        internal MetricsMap<T>.Entry? GetEntry(MetricsMap<T> map) => _entries?.FirstOrDefault(entry => entry.Map == map);
```

Comment:
It should probably be `_entries!` instead I don't think this is supposed to be called before `_entries` is set.